packages/frontend/src/contexts/AppContext.tsx

```
@@ -0,0 +1,147 @@
+  canAddTicket: (passType: PassType, quantity: number) => boolean;
+};
+
+const AppContext = createContext<AppContextType | undefined>(undefined);
+
+export const AppProvider = ({ children }: { children: ReactNode }) => {
+  // Charger le panier depuis le localStorage si présent
+  const getInitialCart = (): Cart => {
+    try {
+      const stored = localStorage.getItem('cart');
+      if (stored) return JSON.parse(stored);
+    } catch (e) {}
```

ESLint

[error] Empty block statement.

(no-empty)